### PR TITLE
Added animations to home screen and eatery logo to login page

### PR DESCRIPTION
**Pull request by JaysonHahn**
## Overview

Added animations to home screen, eatery logo to login page And changes onboarding screens

## Changes Made

- Used hero to translate the image of eatery from `HomeViewController` to `EateryViewController`
- Added in a fade animation in `EateryViewController` for all other views in `EateryViewController`
- Added eatery logo to `ProfileLoginViewController`
- Changes the onboarding screens according to new designs

## Test Coverage

Used manual testing to access the animations works for all different use cases of the animations

## Screenshots (delete if not applicable)

<details>

  <summary>Screen Shot Name</summary>
Animations and eatery logo

  
  

https://github.com/cuappdev/eatery-blue-ios/assets/46629787/a37fd17f-b312-43c0-bbd0-f22217a00264

New Onboarding pages

https://github.com/cuappdev/eatery-blue-ios/assets/46629787/b1bc8010-bbc9-4f3f-a3aa-7c1f97b910a1

</details>

### CODE REVIEW

File: Eatery Blue/UI/EateryScreen/EateryViewController.swift [hunk 1]
```diff
@@ -10,6 +10,7 @@ import EateryModel
 import UIKit
 import CoreLocation
 import MapKit
+import Hero
```
Comment (vinnie4k) [inline Eatery Blue/UI/EateryScreen/EateryViewController.swift:13]: Imports should be sorted alphabetically.

File: Eatery Blue/UI/EateryScreen/EateryViewController.swift [hunk 2]
```diff
@@ -36,6 +39,7 @@ class EateryViewController: UIViewController {
 
         setUpView()
         setUpConstraints()
+        self.hero.isEnabled = true
```
Comment (vinnie4k) [inline Eatery Blue/UI/EateryScreen/EateryViewController.swift:43]: Remove new line.

File: Eatery Blue/UI/EateryScreen/EateryViewController.swift [hunk 3]
```diff
@@ -36,6 +39,7 @@ class EateryViewController: UIViewController {
...
```
Comment (vinnie4k) [inline Eatery Blue/UI/EateryScreen/EateryViewController.swift:42]: Not sure, but I don't think you need `self.`

File: Eatery Blue/UI/Profile/ProfileLoginViewController.swift [hunk 4]
```diff
@@ -121,6 +132,11 @@ class ProfileLoginViewController: UIViewController {
             make.bottom.lessThanOrEqualTo(view.keyboardLayoutGuide.snp.top).offset(-12)
         }
         loginButton.content.setContentCompressionResistancePriority(.required, for: .vertical)
+        
+        eateryLogo.snp.makeConstraints({make in
+            make.centerX.centerY.equalToSuperview()
+            make.width.height.equalTo(view.frame.width - 124)
+        })
```
Comment (vinnie4k) [inline Eatery Blue/UI/Profile/ProfileLoginViewController.swift:139]:
```suggestion
        eateryLogo.snp.makeConstraints { make in
            make.centerX.centerY.equalToSuperview()
            make.width.height.equalTo(view.frame.width - 124)
        }
```

**Review by vinnie4k — APPROVED** [review]
Great work! Just fix those small issues and you should be good. Ping me on Slack when another reviewer approves this PR.

**Review by vinnie4k — APPROVED** [review]
Good to go.

**Review by antoinztte — APPROVED** [review]
🚀

**Pull request merged**